### ISSUE TITLE
Backport: Change the default Prometheus cpu/memory limits and reservation

### ISSUE THREAD

**Issue by loganhz** [rancher/rancher#19350]
https://github.com/rancher/rancher/issues/19349

Change default CPU Limits to 1000
Change default CPU Reservation to 750

Change default Memory Limits to 1000
Change default Memory Reservation to 750

**Comment by alena1108** [comment]
Can be tested with rancher/rancher:v2.2.2-rc2

**Comment by davidnuzik** [comment]
@loganhz @sangeethah the new limits are too high now for a 2vCPU cluster. Is this what we want? Now we are in a situation again where this is less likely to work out of the box without a bigger cluster.
I often test with a 2vCPU cluster but now this is not good enough:

![image](https://user-images.githubusercontent.com/45179589/55574634-b2cd8100-56c1-11e9-8909-c258050e6683.png)

cc: @deniseschannon 

I thought memory was more important than CPU reservation. I think ideally leave prometheus CPU reservation at old 500 value, but keep new 750 MiB memory reservation value.

**Comment by sangeethah** [comment]
@alena1108 @cjellick Wanted to confirm that the CPU/Memory Reservation we have is the ideal settings we want ?

**Comment by davidnuzik** [comment]
Based on discussion with @sangeethah this CPU reservation amount (and memory reservation) are expected. Going forward I will always test with larger clusters or with an additional worker node so I have sufficient vCPU to test.

**Comment by davidnuzik** [comment]
Version: v2.2.2-rc2

Confirmed the new default values:

>Change default CPU Limits to 1000
>Change default CPU Reservation to 750
>
>Change default Memory Limits to 1000
>Change default Memory Reservation to 750

And as pictured:
![image](https://user-images.githubusercontent.com/45179589/55578408-b1548680-56ca-11e9-926e-8380ce532ef1.png)

Validated HTTP (example below):
```
prometheus.resources.core.limits.cpu: "1111m"
prometheus.resources.core.limits.memory: "1212Mi"
prometheus.resources.core.requests.cpu: "770m"
prometheus.resources.core.requests.memory: "777Mi"
```

Confirmed that these changes take effect. The prometheus-cluster-monitoring-0 pod restarts (prometheus container) so the change takes effect:

![image](https://user-images.githubusercontent.com/45179589/55578505-f4165e80-56ca-11e9-9b99-5c48535fdc98.png)

I smoke tested this area. I tried setting rather low specs as well such as cpu reservation 50 cpu limit 75 and monitoring still worked as expected (in my case with my small cluster). 

Note testing new node exporter settings/defaults/limits was already validated in another issue.
